jobs/net-weave/spec

```diff
@@ -24,5 +24,4 @@ properties:
   debug:
     default:     false
     description: Enables verbose debugging, suitable for troubleshooting.
-
```

Comment:
This blank line removal is unnecessary and unrelated.

Reply:
done